src/modules/objectsearch/ObjectSearchComponent.js

```diff
@@ -126,6 +126,7 @@ export class ObjectSearchComponent extends React.Component {
                               />
                             </a>
                           </th>
+                          <th>{Config.isDev && 'Se objekt'}</th>
```

Comment:
I like the pattern but should we take it one step future.
If we add a object in the config where we can add feature toggles. Then we can easily enable/disable them. When we're happy with the feature it is easy to find all the places we need to remove the toggle check. 

You don't need to do this in this PR but I think we should discuss the pattern and decide if we want to use it or not.